docker-compose.yml

```diff
@@ -0,0 +1,145 @@
+version: '3'
+services:
+
+  mysql:
+    container_name: mysql
+    image: amd64/mysql:5.7.37
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+    volumes:
+      - ./mysql/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+      - mysql_data:/var/lib/mysql
+
+  elasticsearch:
+    container_name: elasticsearch
+    hostname: elasticsearch
+    build: ./elasticsearch/.
+    environment:
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "discovery.type=single-node"
+      - xpack.security.enabled=false  # override what is in elasticsearch.yml
+      - bootstrap.memory_lock=true
+      #- ELASTIC_PASSWORD=ElasticRocks  # default username is elastic
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - ./elasticsearch/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+      - es_data:/usr/share/elasticsearch/data
+    ports:
+      - "9200:9200"
+      - "9300:9300"
+
+  kibana:
+    container_name: kibana
+    image: docker.elastic.co/kibana/kibana:7.16.2
+    ports:
+      - 5601:5601
+
+  chorus-ui:
+    container_name: chorus_es_ui
+    build: ./reactivesearch/.
+    volumes:
+      - './reactivesearch:/usr/src/app'
+      - '/usr/src/app/node_modules'
+    ports:
+      - 4001:3000
+
+  redis:
+    container_name: quepid_redis
```

Comment:
Don't need this? If you are doing this why not change the whole `redis` section to `quepid_redis`?

Reply:
changed `container_name` to `redis`